src/CommunityToolkit.Maui/Behaviors/Validators/MultiValidationBehavior.shared.cs

```diff
@@ -91,7 +91,7 @@ void OnChildrenCollectionChanged(object? sender, NotifyCollectionChangedEventArg
 				var isSuccessful = ((ICommunityToolkitBehavior<VisualElement>)child).TrySetBindingContextToAttachedViewBindingContext();
 				if (!isSuccessful)
 				{
-					Trace.WriteLine($"Setting {nameof(BindingContext)} for {child.GetType()} failed");
+					Trace.TraceError($"Setting {nameof(BindingContext)} for {child.GetType()} failed");
```

Comment:
Please do not use `TraceError` in the library. It causes crash reporting tools like sentry.io to report it as an unhandled exception. Leading to our library causing erroneous bug reports for our users.